src/main/kotlin/com/openlattice/linking/controllers/LinkingFeedbackController.kt

```diff
@@ -113,8 +113,10 @@ constructor(
 
     private fun linkingFeedbackCheck(entityDataKeys: Set<EntityDataKey>, linkingEntityDataKey: EntityDataKey) {
         val linkingEntitySet = entitySetManager.getEntitySet(linkingEntityDataKey.entitySetId)!!
-        val entityKeyIdsOfLinkingId = entityDataStore
-                .getEntityKeyIdsOfLinkingIds(setOf(linkingEntityDataKey.entityKeyId)).first().second
+        val entityKeyIdsOfLinkingId = entityDataStore.getEntityKeyIdsOfLinkingIds(
```

Comment:
Would love to get rid of `entityDataStore` and replace with `LinkingQueryService` as this call in `entityDataStore` just wraps the call in `LinkingQueryService`. Beyond the scope of this pr, just food for thought

Reply:
👍